src/fuel_watcher.rs

```diff
@@ -1,163 +1,260 @@
-                                );
-                                actions.action(
-                                    portal_withdraw_alert.alert_action.clone(),
-                                    Some(portal_withdraw_alert.alert_level.clone()),
-                                );
-                            }
-                        }
-                    }
-                }
-
-                // check ERC20 token withdrawals
-                for gateway_withdraw_alert in &watch_config.gateway_withdraw_alerts {
-                    if gateway_withdraw_alert.alert_level != AlertLevel::None {
-                        match fungible_token_contract
-                            .get_amount_withdrawn(
-                                gateway_withdraw_alert.time_frame,
-                                &gateway_withdraw_alert.token_address,
-                            )
-                            .await
-                        {
-                            Ok(amount) => {
-                                let amount_threshold = FuelChain::get_value(
-                                    gateway_withdraw_alert.amount,
-                                    gateway_withdraw_alert.token_decimals,
-                                );
-                                if amount >= amount_threshold {
-                                    alerts.alert(
-                                        format!(
-                                            "ERC20 withdraw threshold of {}{} over {} seconds has been reached. Amount withdrawn: {}{}",
-                                            amount_threshold, gateway_withdraw_alert.token_name, gateway_withdraw_alert.time_frame, amount, gateway_withdraw_alert.token_name
-                                        ),
-                                        gateway_withdraw_alert.alert_level.clone(),
-                                    );
-                                    actions.action(
-                                        gateway_withdraw_alert.alert_action.clone(),
-                                        Some(gateway_withdraw_alert.alert_level.clone()),
-                                    );
-                                }
-                            }
-                            Err(e) => {
-                                alerts.alert(
-                                    format!("Failed to check ERC20 withdrawals: {e}"),
-                                    gateway_withdraw_alert.alert_level.clone(),
-                                );
-                                actions.action(
-                                    gateway_withdraw_alert.alert_action.clone(),
-                                    Some(gateway_withdraw_alert.alert_level.clone()),
-                                );
-                            }
-                        }
-                    }
-                }
+            for _ in 0..POLL_LOGGING_SKIP {
```

Comment:
whats the point of this for loop inside a infinite loop statement?

Reply:
Idk, thats how they had them before but after testing I m just going to remove them as there is no impact.